ibmvmc/ibmvmc.c

```diff
@@ -1723,11 +1780,19 @@ static int ibmvmc_probe(struct vio_dev *vdev, const struct vio_device_id *id)
 	pr_debug("ibmvmc: Probe: liobn 0x%x, riobn 0x%x\n", adapter->liobn,
 			adapter->riobn);
 
+	init_waitqueue_head(&adapter->reset_wait_queue);
+	adapter->reset_task = kthread_run(ibmvmc_reset_task, adapter, "ibmvmc");
+	if (IS_ERR(adapter->reset_task)) {
+		pr_err("ibmvmc: Failed to start reset thread\n");
+		ibmvmc.state = ibmvmc_state_failed;
+		return -1;
```

Comment:
I can't do
`adapter->reset_task = NULL;`
`return PTR_ERR(adapter->reset_task);`
because that would return 0 (success)... I can do this instead:

`rc = PTR_ERR(adapter->reset_task);`
`adapter->reset_task = NULL;`
`return rc;`